apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -545,6 +545,8 @@ public function deleteShare(string $id): DataResponse {
 			$this->lock($share->getNode());
 		} catch (LockedException $e) {
 			throw new OCSNotFoundException($this->l->t('Could not delete share'));
+		} catch (NotFoundException $e) {
+			$this->logger->debug('File of deleted share was not found, skip locking', ['exception' => $e]);
```

Comment:
```suggestion
			$this->logger->debug('File of to be deleted share was not found, skip locking', ['exception' => $e]);
```
Since the share is not yet deleted at this point.